src/test.jest.testRunner.ts

```diff
@@ -79,19 +89,53 @@ async function askRunner(
     ...children,
   ]);
   const askJsonTargetPath = getTargetPath(testPath, 'askjson', '../src');
-  await mkdir(dirname(askJsonTargetPath), { recursive: true });
+  await mkdir(dirname(askJsonTargetPath), {
+    recursive: true,
+  });
   await writeFile(askJsonTargetPath, JSON.stringify(askJson, null, 2), {
     encoding: 'utf-8',
   });
 
+  const askFormatted = prettier.format(source, {
+    parser: 'askscript' as prettier.BuiltInParserName,
+    plugins: [prettierPluginAskScript],
+    // TODO load options from file
+  });
+
+  // const askFormattedPath = getTargetPath(
+  //   testPath,
+  //   'ask.formatted.ask',
+  //   '../src'
+  // );
+  // await mkdir(dirname(askFormattedPath), {
+  //   recursive: true,
+  // });
+  // await writeFile(askFormattedPath, askFormatted, {
+  //   encoding: 'utf-8',
+  // });
+
+  // TODO check that if we format again we get the same thing
+
+  if (process.env.ASK_PRINT_SOURCE) {
+    console.log('\n[PRETTIER]:');
+  }
+
+  source = askFormatted; // use formatted source for testing
```

Comment:
formatted source is processed - this is why some `.askcode` outputs may be different than the unformatted source file